spec/workflow/states/map_spec.rb

```diff
@@ -263,6 +263,15 @@
           end
         end
       end
+
+      context "with both ToleratedFailureCount and ToleratedFailurePercentage" do
+        let(:tolerated_failure_percentage) { 50 }
+        let(:tolerated_failure_count)      { 1 }
```

Comment:
Can you add another spec showing when count might be tolerated but not percentage (i.e. the flipped case to this?)

Reply:
:+1: done, also added a case where they are both present but neither high enough for state success